Source/WebCore/platform/graphics/GraphicsContext.cpp

```diff
@@ -374,7 +374,7 @@ void GraphicsContext::drawPattern(ImageBuffer& image, const FloatRect& destRect,
     image.drawPattern(*this, destRect, tileRect, patternTransform, phase, spacing, options);
 }
 
-void GraphicsContext::drawControlPart(const ControlPart& part, const FloatRect& rect, float deviceScaleFactor, const ControlStyle& style)
+void GraphicsContext::drawControlPart(ControlPart& part, const FloatRect& rect, float deviceScaleFactor, const ControlStyle& style)
```

Comment:
I'd like to find out if we can keep the ControlPart& `const`. Drawing one should not mutate it.